test/cri-containerd/policy_test.go

```diff
@@ -57,10 +59,10 @@ func securityPolicyFromContainers(
 	return base64.StdEncoding.EncodeToString([]byte(policyString)), nil
 }
 
-func sandboxSecurityPolicy(t *testing.T, policyType string, allowEnvironmentVariableDropping bool) string {
```

Comment:
Why remove this test element? Couldn't both be varied, i.e. unencrypted scratch and env dropping?

Reply:
Just looking at the code, it seemed like there was a bad rebase. I don't `sandboxSecurityPolicy` is used anywhere with allow env var dropping... the only places where that parameter is used is in encrypted scratch tests.